BitwardenShared/Core/Platform/Services/ConfigService.swift

```diff
@@ -166,6 +184,7 @@ class DefaultConfigService: ConfigService {
                 responseModel: configResponse
             )
             try? await stateService.setServerConfig(serverConfig)
+            await loadFlags(serverConfig)
```

Comment:
@fedemkr Do you think your [comment](https://github.com/bitwarden/ios/pull/821#discussion_r1715199771) is taken care of in this case?